src/features/content-explorer/content-explorer/ContentExplorerIncludeSubfolders.js

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import Checkbox from '../../../components/checkbox/Checkbox';
+import Tooltip from '../../../components/tooltip';
+import Toggle from '../../../components/toggle';
+
+import messages from '../messages';
+import IconInfo from '../../../icons/general/IconInfo';
```

Comment:
does `InfoBadge16` work? https://opensource.box.com/box-ui-elements/storybook/?path=/story/icon-fill-infobadge16--info-badge-16

Reply:
Is there a version of this in the primary blue color?

Reply:
you need to use CSS to target the fill and path to set to the color.

it would be something like:
```
// inside ContentExplorerIncludeSubfolders.js
<InfoBadge16 className="bdl-ContentExplorerIncludeSubfolders-infoIcon" />

// inside ContentExplorerIncludeSubfolders.scss
.bdl-ContentExplorerIncludeSubfolders-infoIcon {
    path {
        fill: $bdl-box-blue;
    }
}
```